### PR TITLE
Remove generating `cairo-paths` 

### PR DESCRIPTION
**Pull request by kasperski95**
- it doesn't work as expected because package's project root is usually in a subdirectory
- `lib-path` in the configuration file is declared in the `project` section is read directly however lib-path can be passed as argument resulting in weird behavior
- this behavior is undocumented (Do we need to deprecate it first?)
- this feature is confusing, it may work for some packages, but mostly it won't

### CODE REVIEW

File: protostar/compiler/project_cairo_path_builder.py [hunk 1]
```diff
@@ -1,38 +1,18 @@
-import os
 from pathlib import Path
 
-from protostar.configuration_file import ConfigurationFile
-
 
 class ProjectCairoPathBuilder:
```
Comment (MaksymilianDemitraszek) [inline protostar/compiler/project_cairo_path_builder.py:4]: Can we remove this class? It seems too small to abstract it

**Review by MaksymilianDemitraszek — APPROVED** [review]

File: protostar/compiler/project_cairo_path_builder.py (hunk 1 above)
Comment (kasperski95) [reply protostar/compiler/project_cairo_path_builder.py:4]: This logic is used in various places in the application despite being "lite".

Comment (kasperski95) [reply protostar/compiler/project_cairo_path_builder.py:4]: Resolving. I don't want to mix refactors with functionality changes if possible.

Comment (MaksymilianDemitraszek) [reply protostar/compiler/project_cairo_path_builder.py:4]: Can you make a task to remove it?

Comment (kasperski95) [reply protostar/compiler/project_cairo_path_builder.py:4]: https://github.com/software-mansion/protostar/issues/1207

**Pull request merged**